src/constants/remoteConfigConstants.js

```diff
@@ -62,6 +62,7 @@ export const REMOTE_CONFIG = {
   FEATURE_WERT_COMMODITIES: 'feature_wert_commodities',
   FEATURE_WERT_CURRENCY: 'feature_wert_currency',
   FEATURE_WERT_COMMODITY: 'feature_wert_commodity',
+  FIRESTORE_SYNC_URL: 'app_api_user_endpoint',
```

Comment:
Could you change this to `APP_API_USER_ENDPOINT` please and update the references elsewhere, this is just for consistency.